index.html

```diff
@@ -7,8 +7,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
   <!-- bootstrap css -->
-
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
   <!-- custom styles -->
+  <link rel="stylesheet" type="text/css" href="main.css">
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
Put this at the bottom of your `<body>` on line 57. This helps with perceived page load time, because it allows all your HTML code to load before loading JavaScript files.